languages/ar.ini

```diff
@@ -378,6 +377,7 @@ First = "الأول"
 First Name = "الاسم الأول"
 fix_metadata = "أجل، قم بإصلاح الميتاداتا؛سأنتظر"
 for search = "نتيجة بحث عن"
+for_search_html = "for search: '<strong>%%lookfor%%</strong>'"
```

Comment:
It looks like something went wrong with the automated processing of the "for search" string -- the English version is propagated to all of the language files, instead of cloning the native version. Also, is 'for search' able to be deleted after the string generation is corrected?